matrix_sdk_base/src/rooms/normal.rs

```diff
@@ -162,6 +162,10 @@ impl Room {
             .clone()
     }
 
+    pub fn is_public(&self) -> bool {
+        matches!(self.joine_rules(), JoinRule::Public)
+    }
+
     pub fn joine_rules(&self) -> JoinRule {
```

Comment:
Is this supposed to be called `joine_rules` instead of `join_rules`?

Reply:
Hah, tab completion should have spell checking, thanks.